doc/source/security/encrypt-traffic.rst

```diff
@@ -12,6 +12,52 @@ This includes traffic between client applications and cluster nodes,
    :local:
    :depth: 1
 
+There are two approaches to configure encryption: automatic and manual.
+
+.. _ssl-auto-conf:
+
+SSL Automatic Configuration
+===========================
+
+|PXC| includes the :variable:`pxc-encrypt-cluster-traffic` variable
+that enables automatic configuration of SSL encryption.
+By default, it is disabled, meaning that you need to configure SSL manually
+if you want to encrypt SST and IST traffic, and internal communication.
+
+This variable cannot be changed on the command line during runtime.
+To enable automatic configuration of SSL encryption,
+set ``pxc-encrypt-cluster-traffic=ON`` in the the ``[mysqld]`` section of
+the :file:`my.cnf` file, and restart the cluster.
+
+Automatic configuration of the SSL encryption needs key and certificate files.
+Creation of these files is covered by the :ref:`generate-keys-certs` section.
+Necessary key and certificate files are first searched at the ``ssl-ca``,
+``ssl-cert``, and ``ssl-key`` options under ``[mysqld]``.
+If these options are not set, it then looks in the data directory for
+:file:`ca.pem`, :file:`server-cert.pem`, and :file:`server-key.pem` files.
+
+.. note:: The ``[sst]`` section is not searched.
+
+If all three files are found, they are used to configure encryption for SST
+using XtraBackup (with ``encrypt=4`` mode, as explained in :ref:`xtrabackup`
+subsection).
+If any of the files are missing, a fatal error is generated.
+
+The following settings are applied (and overridden if necessary)::
```

Comment:
"The following settings are ..." could be reworded something on this line

"Setting pxc-encrypt-cluster-traffic=ON is like setting following things in my.cnf"